sal_interfaces/ATCamera/ATCamera_Events.xml

```diff
@@ -6464,18 +6464,189 @@
       <Count>1</Count>
     </item>
   </SALEvent>
-  <!--Created by xmlmaker2 for subsystem ATCamera class ATCamera_logevent_bonn_shutter_PeriodicTasks_timersSettingsApplied using level 1 for category timers-->
+  <!--Created by xmlmaker2 for subsystem ATCamera class ATCamera_logevent_bonn_shutter_PeriodicTasks_timersConfiguration using level 1 for category timers-->
   <!--======================================================================================================================================================-->
   <SALEvent>
     <Subsystem>ATCamera</Subsystem>
-    <EFDB_Topic>ATCamera_logevent_bonn_shutter_PeriodicTasks_timersSettingsApplied</EFDB_Topic>
+    <EFDB_Topic>ATCamera_logevent_bonn_shutter_PeriodicTasks_timersConfiguration</EFDB_Topic>
+    <item>
+      <EFDB_Name>version</EFDB_Name>
+      <Description>Version of the settings, formatted using CCS conventions</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>agentmonitorservice_taskPeriodMillis</EFDB_Name>
+      <Description>MISSING</Description>
```

Comment:
Be nice to see all of the `MISSING` descriptions filled in at some point. I understand if it doesn't happen right now.